common.yml

```diff
@@ -343,6 +343,7 @@ functions:
       working_dir: src/github.com/mongodb/mongo-tools
       script: |
         ${_set_shell_env}
+        PATH=$PATH:$HOME
```

Comment:
We need this for the `SAEvergreenValidate` func to run the `evergreen` binary, which is found in the `$HOME` directory on distros.

Reply:
perhaps we can do this in `_set_shell_env` instead?

Reply:
Hmm, I tried simply adding `export PATH=$PATH:$HOME` at the bottom of `_set_shell_env` in [this patch build](https://evergreen.mongodb.com/task/mongo_tools_static_evergreen_validate_patch_cbdf89967b93b03a915e87d1c7694f77671c4769_6070a3f89ccd4e38a8c71257_21_04_09_18_59_11) (with the [common.yml diff](https://evergreen.mongodb.com/filediff/6070a3f89ccd4e38a8c71257/?file_name=common.yml&patch_number=0)), but it's somehow giving me a Go import cycle error.

I also tried it without `export` but it's the [same result](https://evergreen.mongodb.com/task/mongo_tools_static_evergreen_validate_patch_cbdf89967b93b03a915e87d1c7694f77671c4769_6070ad85c9ec440a16248859_21_04_09_19_39_54) (with the [common.yml diff](https://evergreen.mongodb.com/filediff/6070ad85c9ec440a16248859/?file_name=common.yml&patch_number=0)). Not totally sure why that's happening...

I was originally thinking that `_set_shell_env` was more for Evergreen expansions, but I'm fine exporting the PATH there if you have a sense of what could be causing that error.